packages/node/src/BacktraceClient.ts

```diff
@@ -36,19 +39,32 @@ export class BacktraceClient extends BacktraceCoreClient {
             },
             fileSystem: new NodeFileSystem(),
         });
+
+        const breadcrumbsManager = this.modules.get(BreadcrumbsManager);
+        if (breadcrumbsManager && this.sessionFiles) {
+            breadcrumbsManager.setStorage(
+                FileBreadcrumbsStorage.create(this.sessionFiles, options.breadcrumbs?.maximumBreadcrumbs ?? 100),
+            );
+        }
     }
 
     public initialize(): void {
-        super.initialize();
+        const lockId = this.sessionFiles?.lockPreviousSessions();
 
-        this.loadNodeCrashes();
+        try {
```

Comment:
just in case - do you expect any of these method can throw an error? I think it's safe to use them without try/catch, however I also know why you added it here. Maybe it's worth to leave a note about it?

Reply:
I don't care if the method will throw an error or not. I want to always make sure that the locks are released after.